src/test/eventfd.c

```diff
@@ -13,7 +13,9 @@ static void do_test(int fd) {
 }
 
 int main(void) {
+#ifdef SYS_eventfd
   do_test(syscall(SYS_eventfd, 0));
```

Comment:
Seems like it'd be better to just use eventfd2.

Reply:
The libc wrapper on the next line will test that. I think it's good to just have something that calls the old syscall also, just to make sure it doesn't bitrot.